core/src/main/scala/sttp/client3/RequestT.scala

```diff
@@ -246,6 +246,14 @@ case class RequestT[U[_], T, -R](
 
   def autoDecompressionDisabled: Boolean = tags.getOrElse(disableAutoDecompressionKey, false).asInstanceOf[Boolean]
 
+  private val httpVersionKey = "httpVersion"
+
+  // Used as a workaround to keep binary compatibility
+  // TODO: replace with additional parameter in RequestOptions when writing sttp4
+  def httpVersion(version: HttpVersion): RequestT[U, T, R] = tag(httpVersionKey, version)
```

Comment:
it would be good to add some comment on what this does, and that this is not uniformly implemented in all backends (that is, some backends support some versinos, some - none)